lib/samson/hooks.rb

```diff
@@ -63,6 +110,32 @@ def render_views(name, view, *args)
         nil
       end
 
+      def plugin_setup
+        Samson::Hooks.plugins.
+          each(&:require).
+          each(&:precompile_assets).
+          each(&:add_lib_path).
+          each(&:add_decorators)
+      end
+
+      def render_javascripts(view)
+        Samson::Hooks.plugins.map(&:javascripts).flatten.each do |script|
+          view.instance_exec do
+            concat view.javascript_include_tag(view.asset_path(script))
+          end
+        end
+        nil
+      end
+
+      def render_stylesheets(view)
+        Samson::Hooks.plugins.map(&:stylesheets).flatten.each do |stylesheet|
```

Comment:
If I use each, what I pass to ```flatten``` are the plugins objects and what I really what are the ```stylesheets``` (unless I am not interpreting correctly your comment).